inst/base/mrgsolv.h

```diff
@@ -140,4 +140,8 @@ typedef std::vector<double> dvec;
 #define MRGSOLVE_CONFIG_SIGNATURE databox& self, const dvec& _THETA_, const double neq, const double npar
 #define MRGSOLVE_CONFIG_SIGNATURE_N 4
 
+//! signature for <code>$EVENT</code>
+#define MRGSOLVE_EVENT_SIGNATURE const dvec& _A_, const dvec& _A_0_,  dvec& _THETA_,  const dvec& _F_, const dvec& _R_,  databox& self, const dvec& _pred_, dvec& _capture_, mrgsolve::resim& simeps
+#define MRGSOLVE_EVENT_SIGNATURE_N 9
+
```

Comment:
This is important; I'm having `$EVENT` function identical to `$TABLE`. Anything you _could_ do in one, you _could_ do in the other.

Reply:
>  it could ensure they act the same.

I like the idea of sharing for this reason, and I think the confusion could be made less likely by the combination of 1) putting in a code comment and 2) moving the macros next to each other.

Another idea:  what about renaming `MRGSOLVE_TABLE_SIGNATURE`/`MRGSOLVE_TABLE_SIGNATURE_N` to indicate both `TABLE` and `EVENT` and then using it for both?  I suppose `MRGSOLVE_TABLE_OR_EVENT_SIGNATURE` is getting a bit long, but I think it'd resolve both points above.

Reply:
Thanks. I moved the macros next to each other and noted they are the same. I ended up copying the TABLE macro into the EVENT macro. So they will have different names in the code, but guaranteed to be in sync.

Reply:
I'm going to send this off to winbuilder just to make sure the macros are behaving as I think they will on gcc.  If that comes back ok, will merge.